docs/geedocs/docsrc/answer/fusionAdministration/confTaskRulesForFusionPerf.rst

```diff
@@ -344,12 +344,23 @@ Configure Fusion performance
       number of objects. This option is controlled by
       ``LimitMemoryUtilization`` and its default value is 0 (false). The
       maximum memory usage for each cache can be set with
-      ``MaxAssetCacheMemorySize`` and ``MaxVersionCacheMemorySize``.
-      This size is measured in bytes and the default setting is 1GB.
-      While this option is enabled, the caches will not be limited by
-      the number of objects in the cache. It should be noted that as of
-      now the reported memory usage is very low and that this option is
-      not recommended for production environments.
+      ``MaxAssetCacheMemorySize`` and ``MaxVersionCacheMemorySize`` and
+      is in terms of bytes. While this option is enabled, the caches will
+      not be limited by the number of objects in the cache.
+
+      .. _prune-percent:
+
+      Another experimental option in ``misc.xml`` allows the user to limit
+      how often the asset and asset version caches are pruned, which can
+      improve performance. Assets that have been modified can not be 
+      removed from the cache until they are written to disk. To limit
+      pruning, the amount of unwritten modified assets is compared to a 
+      percentage of the current size of each cache. This percentage is
+      controlled by ``PrunePercent`` and is set to 100 by default, which
+      retains current behavior. The value for ``PrunePercent`` can be any
```

Comment:
Minor: "retains current behavior" reads a bit funny to me since the current behavior is to do whatever `PrunePercent` says. You could say "retains previous behavior", but I think it would be better to be more explicit and say something like "prunes the caches as often as possible."

I don't think it's a huge deal, though, so I'm going to go ahead and approve.